Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css

```diff
@@ -1,374 +1,207 @@
+ul.DropDownMenuOptions li a
 {
-  border-left: solid 1px #AAAAAA;
+  cursor: pointer;
+  text-decoration: none;
+  display: flex;
+  padding: var(--remotion-themed-padding);
+  white-space: nowrap;
+  color: Black;
+  outline: none;
 }
 
-span.DropDownMenuSelect:active a.DropDownMenuButton
+ul.DropDownMenuOptions li a span
 {
-  border-left: 1px solid #569CE0;
+  display: inline-flex;
+  margin-left: var(--remotion-themed-margin);
 }
 
-span.DropDownMenuSelect.nestedHover:active a.DropDownMenuButton
+ul.DropDownMenuOptions li a img
 {
-  border-left: 1px solid #AAAAAA;
+  display: inline-flex;
+  margin-right: var(--remotion-themed-margin);
 }
 
-span.DropDownMenuSelect
+span.DropDownMenuSelect:hover,
+span.DropDownMenuSelect:active
 {
-  background-image: url(../Image/sprite.svg#ButtonBackground);
-  background-size: 1px 20px;
-  background-color: #E6E5E7;
-  background-position: center top;
-  background-repeat: repeat-x;
+  border: var(--remotion-themed-border);
 }
 
+/* The focus is immediately lost after moving the mouse over
+   the pop-up due to it only being a kind of DIY select. */
 span.DropDownMenuSelect.focus,
-span.DropDownMenuSelect:hover
-{
-  color: Black;
-  background-image: url(../Image/sprite.svg#ButtonBackgroundHover);
-  background-size: 1px 20px;
-  background-color: #DEECFC;
-  border: 1px solid #7CB2E6;
-}
-
-span.DropDownMenuSelect.nestedHover:hover
-{
-  background-image: url(../Image/sprite.svg#ButtonBackground);
-  background-size: 1px 20px;
-  background-color: #E6E5E7;
-  border: solid 1px #AAAAAA;
-}
-
-span.DropDownMenuSelect:active
+span.DropDownMenuSelect:focus-within
 {
-  background-image: url(../Image/sprite.svg#ButtonBackgroundActive);
-  background-size: 1px 20px;
-  background-color: #C5E0FC;
-  border: 1px solid #569CE0;
+  background-color: var(--color-highlight);
+  border: var(--remotion-themed-border);
 }
 
-span.DropDownMenuSelect.disabled:hover
+tr.bocListDataRow span.DropDownMenuContainer,
+tr.bocListDataRowSelected span.DropDownMenuContainer
 {
-  background-image: url(../Image/sprite.svg#ButtonBackground);
-  background-size: 1px 20px;
-  background-color: #ECEEEC;
-  border: solid 1px #BCBCBC;
+  display: inline-flex;
 }
 
-span.DropDownMenuSelect a.DropDownMenuButton:first-child
+td.buttonCell span.DropDownMenuSelect
 {
-  width: 100%;
-  text-align: center;
+  /* only when display just arrow */
+  display: inline-flex;
 }
 
-span.DropDownMenuSelect a.DropDownMenuButton:focus
+span.DropDownMenuSelect a.DropDownMenuLabel
 {
+  position: relative;
+  height: var(--remotion-themed-line-height);
+  padding: var(--remotion-themed-padding);
+  border-style: none;
+  color: Black;
   outline: none;
 }
 
-span.DropDownMenuSelect.disabled a.DropDownMenuButton
+span.DropDownMenuSelect a.DropDownMenuButton
 {
-  border-left: solid 1px #BCBCBC;
+  position: absolute;
+  inset: 0 0 0 auto; /* shorthand for positioning */
+  border-left: var(--remotion-themed-border);
```

Comment:
Didn't search for the precise border declaration but the border between the button and the text should be a lighter gray.
And there's the layout option of displying just the dropdown-arrow without text. You can find a live sample in SecurityManager AXL Editor or configure one of the existing dropdown-menus.

And please drop the boclist-references, will see which one we will need.